File: net.tenteCsharp/src_code/heaterManagement/FloorGUI.java


cclass heaterManagement.HeaterManagement; 

public cclass FloorGUI extends TypeComponent{ 

	public HeaterNotifyPort heaterNotifyPort;
	
	
	public FloorGUI (String id){
		super(id);
	}
	
	public HeaterNotifyPort getHeaterNotifyPort(){
		return heaterNotifyPort;
	}
	
	public void setHeaterNotifyPort(FloorGUI.HeaterNotifyPort value){
		this.heaterNotifyPort=value;
	}
	

	
	
	public cclass HeaterNotifyPort extends TypePort{

		public ArrayList portsIGeneralHeaterNotify;

		public HeaterNotifyPort (){
			super();
			portsIGeneralHeaterNotify=new ArrayList();

		}


			
		public void connectPort(IGeneralHeaterNotify port){
			portsIGeneralHeaterNotify.add(port);
		}
	}

}

File: trunk/Elio/pseudoCodeGeneratorElio/src-gen/heaterManagement/FloorGUI.cs
using System;
using System.Collections;
using System.Text;

namespace SmartHome
{
	public partial class FloorGUI : TypeComponent
	{
		public HeaterNotifyPort heaterNotifyPort;
	
		public FloorGUI (String id) 
			: base(id)
		{
		}
		public HeaterNotifyPort getHeaterNotifyPort()
		{
			return heaterNotifyPort;
		}
	
		public void setHeaterNotifyPort(FloorGUI.HeaterNotifyPort value)
		{
			this.heaterNotifyPort=value;
		}
	

	
		
		public class HeaterNotifyPort : TypePort
		{
 		public ArrayList portsIGeneralHeaterNotify = new ArrayList();

			public HeaterNotifyPort()
				: base()
			{
				portsIGeneralHeaterNotify=new ArrayList();

			}


			public ArrayList getPortsIGeneralHeaterNotify()
			{
	        	return portsIGeneralHeaterNotify;
	        }
				
			public void connectPort(IGeneralHeaterNotify port)
			{
				portsIGeneralHeaterNotify.Add(port);
			}
	
		}
	}
}